plugins/techdocs-backend/src/helpers.ts

```diff
@@ -124,10 +125,12 @@ export const checkoutGitRepository = async (
   const user =
     process.env.GITHUB_PRIVATE_TOKEN_USER ||
     process.env.GITLAB_PRIVATE_TOKEN_USER ||
+    process.env.AZURE_PRIVATE_TOKEN_USER ||
```

Comment:
For Azure DevOps there is no need for a specific user, it can be anything. Added this mostly for symmetry.